### ISSUE TITLE
F5 not continuing the script after stopping at a breakpoint

### ISSUE THREAD

**Issue by DarkLite1** [PowerShell/vscode-powershell#2212]
System Details Output
```
### VSCode version: 1.39.0-insider 817eb6b0c720a4ecbc13c020afbbebfed667aa09 x64

### VSCode extensions:
ban.spellright@3.0.46
ms-vscode.powershell@2019.9.0
ms-vscode.powershell-preview@2019.9.0
ritwickdey.LiveServer@5.6.1


### PSES version: 2.0.0.0

### PowerShell version:

Name                           Value
----                           -----
PSVersion                      5.1.14409.1018
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14409.1018
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### Issue Description

I am experiencing a problem with debugging. When a breakpoint is set at a specific line in the PowerShell script and `F5` is pressed the script is executed until that point. So far all good. However, when I want the script to continue after the breakpoint, usually I press `F5` again and it continues to the next breakpoint or until the end if there are no breakpoints anymore.

When pressing `F5` after the latest update it appears that the script does not continue and a pop up at the bottom right corner is shown. 

![image](https://user-images.githubusercontent.com/10828570/65578033-56af2f80-df75-11e9-8b7b-c05d72b9186d.png)

The only way to continue the script, as before, is pressing the blue play button at the top. 

#### Expected Behaviour

Have the `F5` button continue the script after a breakpoint, just like it was before.

**Comment by TylerLeonhardt** [comment]
@DarkLite1 the F5 keybinding is created by VS Code itself. Can you open this issue over in https://github.com/Microsoft/vscode ?

**Comment by corbob** [comment]
> @DarkLite1 the F5 keybinding is created by VS Code itself. Can you open this issue over in https://github.com/Microsoft/vscode ?

Is it though? I am certain I used F5 to continue all the time... I haven't debugged anything since 2019.9.0, but I am 80% sure I did in 2019.5.x...

Is this possibly an issue with these extensions together:
```
ms-vscode.powershell@2019.9.0
ms-vscode.powershell-preview@2019.9.0
```

**Comment by TylerLeonhardt** [comment]
maybe... @DarkLite1 do you have both extensions active? You should have 1 disabled at all times.

**Comment by corbob** [comment]
**TL;DR**: Corbob rambles and screenshots things. Then discovers it's a vscode-insider bug that is fixed in the next update. Link at the end.

I could also be wrong on that...

I don't get the error when I try to F5 again... but it doesn't continue. Probably a keybinding thing:
![Debug menu in vscode. Lack of keybinding for Continue highlighted](https://user-images.githubusercontent.com/30301021/65629303-f5448c00-df87-11e9-866e-2bb4d6af1b4e.png)

~~I have reverted to 2019.5.0 and happy to report that I was incorrect, F5 did not work there either. I think my confusion comes from that I usually am stepping and then just hit continue when F5 doesn't work. But I agree with @DarkLite1 F5 should continue at that point. But also agree with @TylerLeonhardt this to me is a vscode issue as it's the debug keybinding that should likely apply to all languages and not just PowerShell.~~ I typed all this up, then looked at versions of vscode, so I'll leave this here, but I will now also show this screenshot from vscode stable:
![debug menu from vscode stable. Continue has a keyboard shortcut](https://user-images.githubusercontent.com/30301021/65630158-b0b9f000-df89-11e9-8dc3-6741eed26213.png)

So this is definitely an issue in code, and is only present in insiders. The keybinding is also there in insider:
![Debug: Continue keybinding showing it is set to F5 when inDebugMode](https://user-images.githubusercontent.com/30301021/65630197-c7f8dd80-df89-11e9-91da-73fb91e6ec06.png)

Looks like it's reported and fixed: https://github.com/microsoft/vscode/issues/81373

**Comment by TylerLeonhardt** [comment]
Thanks @corbob! I'll close this as external
